keyboards/polykybd/tools/status_oled_preview.py

```
@@ -121,36 +242,76 @@ def draw_bitmap(setpix, data, ox, oy, w=16, h=16):
+                cx += g['xa']
+            draw_bitmap(setp, SUPER2_BMP, name_x + hi + 2, 18, SUPER2_W, SUPER2_H)
+        draw_speed_gauge(setp, COL_X, speed)
+        draw(setp, small, TEXT_X, ROW3, s(hue_name(hue, sat)))
+        draw_right(setp, small, TEXT_R - 4, ROW3, s(str(hue_to_degrees(hue))))
+        draw_bitmap(setp, DEGREE_BMP, TEXT_R - 2, 34, DEGREE_W, DEGREE_H)
+        draw_bitmap(setp, DROPLET_BMP, TEXT_X, DROPLET_Y, DROPLET_W, DROPLET_H)
+        draw(setp, small, TEXT_X + DROPLET_W + SV_ICON_GAP, ROW4,
+             s('%d%%' % byte_to_percent(sat)))
+        vtxt = s('%d%%' % byte_to_percent(val))
+        f, _bm, gl = small
+        hi = 0; cx = 0
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)